src/main/java/org/mskcc/cbio/oncokb/domain/enumeration/EmailSubject.java

```diff
@@ -0,0 +1,9 @@
+package org.mskcc.cbio.oncokb.domain.enumeration;
+
+/**
+ * The EmailSubject enumeration.
+ */
+public enum EmailSubject {
```

Comment:
Here as well. We need details on these categories.

Reply:
Right, will add header comments to both of these.